luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -213,24 +266,35 @@ public ResourceDictionaries getDictionaries() {
          * @param resourceDictionaries  a class that contains resource dictionaries including
          * PhysicalTableDictionary, DimensionDictionary, etc.
          */
-        public Builder(LuthierResourceDictionaries resourceDictionaries) {
-            this.resourceDictionaries = resourceDictionaries;
-            dimensionFactories = getDefaultDimensionFactories();
-            keyValueStoreFactories = getDefaultKeyValueStoreFactories();
-            searchProviderFactories = getDefaultSearchProviderFactories();
-            physicalTableFactories = getDefaultPhysicalTableFactories();
+        public Builder(
+                LuthierResourceDictionaries resourceDictionaries
+        ) {
+            this(
+                    resourceDictionaries,
+                    StandardGranularityParser.getDefaultGrainMap()
+            );
         }
 
         /**
          * Constructor.
-         * <p>
-         * Default to use an empty resource dictionary.
+         *
+         * @param resourceDictionaries  a class that contains resource dictionaries including
+         * PhysicalTableDictionary, DimensionDictionary, etc.
+         * @param granularityDictionary  a named map of all supported granularities
          */
-        public Builder() {
-            this(new LuthierResourceDictionaries());
+        public Builder(
+                LuthierResourceDictionaries resourceDictionaries,
+                GranularityDictionary granularityDictionary
+        ) {
```

Comment:
This doesn't need to be wrapped (our maximum line length is 120 characters, and this is 119.

Reply:
Close one! I was most likely intimidated by the IntelliJ annotation

Reply:
Got it.